src/types/Entity.ts

```diff
@@ -1,3 +1,3 @@
-export interface Entity<T extends number | string = number> {
+export interface Entity<T extends number | string = string | number> {
```

Comment:
I don't see any `Entity` extends without specified type in this PR, which makes me a bit confused about this change.

Do we even have `string` IDs anywhere? 🤔 
I was under impression that all `id` fields are numbers, with strings left for `uuid`.

Reply:
@oleg-semyonov Entity<string> is relied on in the main app. Also, `EmailRecipient` is de facto extending `Entity<string>`, though this is not reflected in the declaration (I'll add it, to be clear).

Thanks!

Reply:
90212f8f53407bda0c92b022dff8994351e3d541

Reply:
Hm... I'm now thinking maybe we don't even need `Entity` type in this repository. It's not adding anything to *this* codebase.

Reply:
> I'm now thinking maybe we don't even need Entity type in this repository. It's not adding anything to this codebase.

Dropped with 85323871ac9888a597cc9a9e67fa41597ff8ff34